source/euler/hyperbolic_system.h

```diff
@@ -1313,5 +1339,159 @@ namespace ryujin
       return result;
     }
 
+
+    template<int dim, typename Number>
+    void HyperbolicSystem::View<dim, Number>::post_process(const vector_type &U,
+                                                           Number t,
+                                                           MPI_Comm mpi_communicator_,
+                                                           const ryujin::OfflineData<dim, Number>& offline_data_)
+    const
```

Comment:
The `const` typically comes at the end of the function declaration, after the closing parenthesis.